### PR TITLE
chore: release 0.46.0

### PR DESCRIPTION
**Pull request by ThibaultFy**

### CODE REVIEW

File: substrafl/__version__.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = "0.45.0"
+__version__ = "0.46.0"
```
Comment (thbcmlowk) [inline substrafl/__version__.py:1]: Would not:
```suggestion
__version__ = "0.45.1"
```
Be more appropriate?

Comment (ThibaultFy) [reply substrafl/__version__.py:1]: Mmmmh it's a minor change so it could enter in the minor definition of semver.

Comment (thbcmlowk) [reply substrafl/__version__.py:1]: I would argue that this is more a security patch transparent for the user than it is a new functionality but this is no big deal

Comment (ThibaultFy) [reply substrafl/__version__.py:1]: Yes I get what you mean... I'm arguing against changing the version il all PRs ahah 😇

Comment (thbcmlowk) [reply substrafl/__version__.py:1]: Fair point, that's why it has been approved beforehand 😃

**Pull request merged**